### ISSUE TITLE
Option to reveal Level and XP bar on level change

### ISSUE THREAD

**Issue by AstudilloMarillo** [Crendgrim/AutoHUD#98]
I like to keep my HUD hidden as much as possible, but I haven't found a great way to do this for the level and XP bar yet.  I don't like linking the level to the hotbar as to me the link is not that logical. Likewise, I don't really need to see the experience bar/level every time I pick up some experience orbs.

The only time I'd really want to see my level is when it changes. So when I gain a level by receiving experience or when I lose levels on enchanting/anvil. It'd be great to have an option to reveal the level and experience bar only when the level changes (maybe separate those in 2 options, so you could choose to only show the level and not the experience bar).

_btw, this mod together with dynamic crosshair are some of my favourite mods I play with, and I just want to thank you for making it and giving us a more immersive playing experience <3_

**Comment by Crendgrim** [comment]
That is how it should work by default, gaining experience should reveal the experience bar. The linking to hotbar is just another trigger to reveal. If this doesn't work, it's a bug. What version of Minecraft and which mod loader are you on?

**Comment by Crendgrim** [comment]
Oh, I misunderstood at first. I see, that's a good idea.

**Comment by Crendgrim** [comment]
What seemed like a trivial change on Fabric then took quite some work on forge-based launchers. This is now implemented in AutoHUD 6.6 :)

**Comment by AstudilloMarillo** [comment]
> What seemed like a trivial change on Fabric then took quite some work on forge-based launchers. This is now implemented in AutoHUD 6.6 :)

It works, thanks so much! 

Now, I hope I'm not pushing my luck with this, but there's one thing that'd make it better for me. I kind of assumed the level and experience bar would be connected, but now I see they are separate. The level looks a little out of place to me without the experience bar, so the same option 'Reveal on level change' for the experience bar would complete this.

**Comment by Crendgrim** [comment]
But when the level changes, the experience bar will be nearly empty always, right? I can add that, though.

**Comment by AstudilloMarillo** [comment]
> But when the level changes, the experience bar will be nearly empty always, right? I can add that, though.

That is true, but to me, the experience bar and level seem interconnected and look awkward by themselves. Maybe it's just me. Would love it if you'd add this though (:

**Comment by Crendgrim** [comment]
That means I'll have to actually think about the config. Was hoping I could just skip that, heh.

**Comment by AstudilloMarillo** [comment]
> That means I'll have to actually think about the config. Was hoping I could just skip that, heh.

I can see that being a thing now, sorry for that, haha!

**Comment by AstudilloMarillo** [comment]
Any update on this? No rush, just curious!
